### PR TITLE
Adding [YALN] Y Alignment custom Axis 

### PR DESCRIPTION
**Pull request by vv-monsalve**
PR to follow up on the inclusion of the new custom axis `YALN Y Alignment` #106 
Required for the Wavefont.

### CODE REVIEW

**Review by davelab6 — APPROVED** [review]

**Comment by vv-monsalve** [comment]
The pipeline for the axis registration has been unblocked. However, now that axis+font should move simultaneously through the pipeline, we should wait until Kablammo and Geologica are merged before merging new axes.

**Comment by ivannggnnavi** [comment]
Bebe

**Pull request merged**
